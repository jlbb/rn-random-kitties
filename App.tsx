import React from "react";
import { Provider } from "react-redux";

import { store } from "./store";
import MainAppScreen from "./screens/MainAppScreen";

const App = () => {
  return (
    <Provider store={store}>
      <MainAppScreen />
    </Provider>
  );
};

export default App;
