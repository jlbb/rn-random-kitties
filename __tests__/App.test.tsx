import App from "../App";
import { shallow } from "enzyme";

describe("RandomKitties App", () => {
  it("renders correctly", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
