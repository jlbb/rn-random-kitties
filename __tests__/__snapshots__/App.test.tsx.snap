// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RandomKitties App renders correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <MainAppScreen />
</Provider>
`;
